[2025-03-12T20:32:41.833+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-12T20:32:41.853+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.featur_engineer scheduled__2025-03-09T13:30:00+00:00 [queued]>
[2025-03-12T20:32:41.858+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.featur_engineer scheduled__2025-03-09T13:30:00+00:00 [queued]>
[2025-03-12T20:32:41.859+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-03-12T20:32:41.865+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): featur_engineer> on 2025-03-09 13:30:00+00:00
[2025-03-12T20:32:41.881+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=262) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-12T20:32:41.884+0000] {standard_task_runner.py:64} INFO - Started process 300 to run task
[2025-03-12T20:32:41.884+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'featur_engineer', 'scheduled__2025-03-09T13:30:00+00:00', '--job-id', '1616', '--raw', '--subdir', 'DAGS_FOLDER/Stock_pred.py', '--cfg-path', '/tmp/tmpi5c_5kbn']
[2025-03-12T20:32:41.885+0000] {standard_task_runner.py:91} INFO - Job 1616: Subtask featur_engineer
[2025-03-12T20:32:41.928+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.featur_engineer scheduled__2025-03-09T13:30:00+00:00 [running]> on host b7aba691e1ff
[2025-03-12T20:32:42.006+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='featur_engineer' AIRFLOW_CTX_EXECUTION_DATE='2025-03-09T13:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-09T13:30:00+00:00'
[2025-03-12T20:32:42.007+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-12T20:32:42.568+0000] {warnings.py:112} WARNING - /opt/***/dags/Stock_pred.py:237: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  input.drop(columns=["Target","CLOSE_t+1"],inplace=True)

[2025-03-12T20:32:42.586+0000] {warnings.py:112} WARNING - /opt/***/dags/Stock_pred.py:240: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  input['CLOSE_diff1'] = input['CLOSE'].diff(1)

[2025-03-12T20:32:42.593+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-12T20:32:42.594+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-12T20:32:42.601+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=featur_engineer, run_id=scheduled__2025-03-09T13:30:00+00:00, execution_date=20250309T133000, start_date=20250312T203241, end_date=20250312T203242
[2025-03-12T20:32:42.643+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-03-12T20:32:42.658+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-12T20:32:42.660+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
